tests/testthat/test-obj-sum.R

```diff
@@ -19,7 +19,7 @@ test_that("data frames and common data vectors have size summary", {
     obj_sum <- obj_sum(x)
     ptype <- type_sum(x)
 
-    expect_equal(obj_sum, !! paste0(ptype, size_sum(x)))
+    expect_equal(obj_sum, !! paste_with_space_if_needed(ptype, size_sum(x)))
```

Comment:
Somewhere in this `test-obj-sum.R` file, I would probably also add a test that shows the result of calling `obj_sum()` on an object with a custom `size_sum()` method.